core/network/backend/src/main/java/com/appcoins/wallet/core/network/backend/HandleApi.kt

```diff
@@ -0,0 +1,25 @@
+package com.appcoins.wallet.core.network.backend
```

Comment:
assuming that this could be here to stay, even though we need to discuss it further, shouldnt this be in the network base module instead of the backend module?

Reply:
Yes, I will move it